solutions/projectstaffing/deployment/arm/scripts/install.py

```diff
@@ -123,9 +123,11 @@ def execute_user_prompts(deployment_name: str, install_config: InstallConfigurat
                                        validate_default_params=validate_default_params)
     install_config.prompt_airtable_config(deployment_name=deployment_name)
     if install_config.sql_auth:
-        # let's generate service password without prompting, it will be saved in key vaults
-        install_config.gdc_service_db_user_password = make_strong_password(length=12)
-        install_config.jgraph_db_user_password = make_strong_password(length=12)
+        # Generating service password without prompting, it will be saved in Az Key Vault.
```

Comment:
Isn't there an equivalent change that needs to be done for Watercooler as well?

Reply:
Yes, there is. Good catch! Fixed!